app/Http/Controllers/Connections/ArclightController.php

```diff
@@ -31,13 +32,22 @@ public function index()
                 $query->where('iso', $iso);
             })->where('source', 'arclight')->select('code')->first())->code;
             if (!$language_id) {
-                return $this->setStatusCode(404)->replyWithError(trans('api.languages_errors_404'));
+                return $this
+                    ->setStatusCode(HttpResponse::HTTP_NOT_FOUND)
+                    ->replyWithError(trans('api.languages_errors_404'));
             }
 
             $components = $this->fetchArclight('media-components/', $language_id, true);
+
+            if (!isset($components->mediaComponents)) {
+                return  $this
+                    ->setStatusCode($this->statusCode)
```

Comment:
we don't want to return any 5xx error here... the data isn't present. HTTP 409 seems closest to what we want to communicate

Reply:
@bradflood , It has been updated in the following line:

https://github.com/faithcomesbyhearing/dbp/pull/867/files#diff-050f24b8934bfb79240c255c161a6789b43ed28255f2e543004fa8d47d8dad78R36